runtime/sema/return_info.go

```diff
@@ -21,7 +21,9 @@ package sema
 type ReturnInfo struct {
 	MaybeReturned      bool
 	DefinitelyReturned bool
+	MaybeJumped        bool
```

Comment:
Is it now possible to merge `xxxReturned` and `xxxJumped` to a single flag?
Are there any use-cases of each being used individually? 🤔

Reply:
Maybe, I'll have a look 👍

Reply:
Merged `MaybeReturned` and `MaybeJumped` in 3021117. `DefinitelyReturned` is used in `Checker.checkFunctionExits` independently to determine if a function with a return type also has a return statement, so `DefinitelyReturned` and `DefinitelyJumped` cannot be merged

Reply:
I see. Thank you for checking!